repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -267,4 +267,5 @@ def _get_firewall_status(service_name):
     return FirewallsFacts(
         firewalld=_get_firewall_status('firewalld'),
         iptables=_get_firewall_status('iptables'),
+        ip6tables=_get_firewall_status('ip6tables'),
```

Comment:
You will have to update the model to make this work

Reply:
Thanks a lot. Totally forgot about model